README.rst

```diff
@@ -20,4 +20,98 @@ Python implementation of PVL (Parameter Value Language)
 Features
 --------
 
-* TODO
\ No newline at end of file
+* TODO
```

Comment:
We need to explain here what PVL is and what label types are compatible with it.  I have tried to read the specification here: http://public.ccsds.org/publications/archive/647x2b1.pdf to figure that out but its not jumping out at me.  I am not going to hold this PR up for this, but we should figure this out.  @wtolson do you know anything offhand?